sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -924,7 +908,9 @@ synchronized void completeWorkflow() {
         mUpdateDialog = null;
         mUpdateSetupFailedDialog = null;
         mUnknownSourcesDialog = null;
-        hideProgressDialog();
+        if (mReleaseDownloaderListener != null) {
+            mReleaseDownloaderListener.hideProgressDialog();
```

Comment:
this should be set to null, next time we want to recreate the object by passing a new mReleaseDetails, while here it will be recycled. If you back back back and start the app from launch menu again, we could get a different release from API call. It's set to null in cancelPreviousTasks, why not here?

Reply:
@guperrot Actually, setting `null` here is not right way to handle this situation. More than, it won't work. Revert setting `null` here and handle change `mReleaseDetails` properly...